### PR TITLE
 Updating delfin support matrix information

### PR DESCRIPTION
**Pull request by NajmudheenCT**
**What type of PR is this?**

> /kind documentation

**What this PR does / why we need it**:

1. Adding enhanced resource  resource support information for all existing drivers 
2. Adding Netapp Cmode support information
**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
NA

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: content/support-matrix/delfin/_index.md [hunk 1]
```diff
@@ -10,17 +10,18 @@ tags: ["support matrix", "delfin", "drivers"]
...
+| Simulated Sample Driver (for testing the framework) | NA                        | NA                                 | NA                      | NA      | √                        | √            | √   | √          | √    | √    | √     | √     | √           | √     | √                    | √         | √           | √               | √                   | 1\. Used only for framework testing purpose                                                                                                                                                             |   |                                                           |                                                          |
```
Comment (sushanthakumar) [inline content/support-matrix/delfin/_index.md:24]: At the end of page, we still have below statement
"Support matrix of drivers for Dell EMC VNX/VPLEX/Unity, IBM Storwize/SVC, Hitachi VSP, HPE 3PAR are currently not updated here."

Comment (NajmudheenCT) [reply content/support-matrix/delfin/_index.md:24]: right , we have separate session for driver level updates, there we do not have these driver specific details of support I will currently remove it . we have only page for vmax and oceanstor

**Comment by sushanthakumar** [comment]
LGTM

**Review by kumarashit — APPROVED** [review]
LGTM

File: content/support-matrix/delfin/_index.md [hunk 2]
```diff
@@ -10,19 +10,20 @@ tags: ["support matrix", "delfin", "drivers"]
...
-| Simulated Sample Driver (for testing the framework) | NA                        | NA                                 | NA                      | NA      | √                        | √                        | √               | √                    | √         | √           | √               | √                   | 1\. Used only for framework testing purpose<br>2\. Performance metric collection is only Alpha version and supports only device level metrics .                                                            |                                                          |
+| vendor                                              | model                     | Version ( Device Manager )         | Access Mechanism (IPV4) | Type    | Basic Device Information | Storage Pool | LUN | Controller | Port | Disk | Qtree | Quota | File system | Share | Capacity Information | SNMP Trap | Alarm Query | Alarm Clearance | Performance Metrics | Remarks/Constraints                                                                                                                                                                                     |   |
```
Comment (anvithks) [inline content/support-matrix/delfin/_index.md:13]: There is an extra column at the end of the table after the `Remarks / Constraints` column. May not be needed. Please check

**Review by anvithks — APPROVED** [review]
LGTM

**Pull request merged**